app/javascript/components/articleCard.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Card, CardTitle, CardBody, CardFooter, Text, TextVariants } from '@patternfly/react-core';
+import '@patternfly/react-core/dist/styles/base.css';
+import colors from '../utils/colors';
+const styles = {
```

Comment:
in Foreman we are usually creating a .scss file in the same component directory and importing the styles in the component .js file to reduce its size.